### PR TITLE
paplayer: interims fix for speed while paused

### PR DESCRIPTION
**Pull request by FernetMenta**
@rechi @Tolriq this fixes the speed issue for paplayer

### CODE REVIEW

**Comment by FernetMenta** [comment]
jenkins build this please

File: xbmc/cores/paplayer/PAPlayer.cpp [hunk 1]
```diff
@@ -949,6 +949,11 @@ void PAPlayer::SetSpeed(float speed)
 
 float PAPlayer::GetSpeed()
 {
+  // maximum crap that pause is handled independent from speed
+  // TODO: change this
```
Comment (MartijnKaijser) [inline xbmc/cores/paplayer/PAPlayer.cpp]: could you use doxygen style todo so it shows up in the todo list?

**Comment by FernetMenta** [comment]
@MartijnKaijser ok this way? I am no doxy expert

**Comment by MartijnKaijser** [comment]
`//! @todo Change this`
Maybe extend it a bit more

**Comment by FernetMenta** [comment]
doxy updated

**Comment by Rechi** [comment]
@FernetMenta since this PR `PlayerControl(Play)` doesn't resume playback for `PAPlayer`, only pausing.
Resuming with the keyboard still works, but not in the GUI and web interface.

**Pull request merged**